Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: Helm Chart
+name: helm-starter
+version: 0.1.0
+appVersion: 0.0.1
+home: https://github.com/cloudposse/helm-starter
+icon: https://raw.githubusercontent.com/chartmuseum/chartmuseum/master/logo.png
```

Comment:
Can you add this to the repo? https://cloudposse.com/wp-content/uploads/sites/29/2017/07/CloudPosse2-TRANSAPRENT.png as `logo.png`. Then link to the raw version of that.